### ISSUE TITLE
Api returns 403 Forbidden

### ISSUE THREAD

**Issue by danito** [sdebruyn/my-bpost-api#7]
Hi,
while trying to install the HACS integration for home-assistant, I got an error:
````python
iohttp.client_exceptions.ClientResponseError: 403, message='Forbidden', url=URL('https://mbassapi.bpost.be/prod_v2/users/emaillogin')
````
I'm afraid the default x-api-key is not valid anymore.
Any idea how to obtain a new one?

**Comment by sdebruyn** [comment]
I don't think it will be the API key since it's the one that is used in the Android app and that hasn't received an update in more than 30 days.
